source/_patterns/01-elements/link/link.scss

```diff
@@ -1,28 +1,33 @@
 @use "@db-ui/base/build/scss/variables" as *;
+@use "@db-ui/base/build/scss/scaling-placeholder" as *;
 @use "@db-ui/base/build/scss/helpers/functions" as *;
 @use "@db-ui/base/build/scss/icon/icons.helpers" as *;
 
 @import "../../../css/partials.meta";
-@import "link.variables";
 
 .elm-link {
-	border-radius: to-rem($pxValue: 4);
+	color: var(--db-current-color, $db-colors-neutral-on-bg-enabled);
 	display: inline-block;
+	border-radius: to-rem($pxValue: 6);
 
 	// Link underline
-	text-underline-offset: to-rem($pxValue: 3);
+	text-underline-offset: to-rem($pxValue: 1);
 	text-underline-position: under;
+	text-decoration-thickness: to-rem($pxValue: 1);
 	@supports (text-underline-offset: 3px) {
 		text-underline-position: auto;
 	}
 
-	&:active {
-		color: initial;
+	// Changing color on hover should be animated by a transition
+	transition: $db-transition-color;
```

Comment:
I'm wondering whether we should probably include the transition property (`color`) in here as well (and/or in DB Base code already).

Reply:
due to the fact that color could be background-color or color, we have to add the color property here. I would suggest to open a new issue for refactoring those transition functions. Is this fine for you @mfranzke ?

Reply:
Agreed, I've just opened a ticket.